_pytest/doctest.py

```diff
@@ -145,7 +143,6 @@ def get_optionflags(parent):
 
 
 class DoctestTextfile(DoctestItem, pytest.Module):
-
```

Comment:
Hmm how did I miss reading that in pep8?
I probably read it and ignored it due to how strange the code looks with extra lines in vim with folding. My bad. Fixed.